components/x-gift-article/bower.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@financial-times/x-gift-article",
+  "description": "",
+  "main": "dist/GiftArticle.es5.js",
+  "private": true,
+  "dependencies": {
+    "o-buttons": "^6.0.0",
+    "o-loading": "^4.0.0"
+  }
```

Comment:
I think it may be feasible to make these npm dev dependencies and do away with Bower entirely for this component 🤔

Reply:
Hi Matt, sorry, could we have a chat about that, I'm not sure I understand this approach

Reply:
As discussed, let's not optimise this right now, as we would need to also update the JavaScript (which is being included onto the page)